### DISCUSSION TITLE
Does anyone know how many unique visitors use Scholia?

### DISCUSSION THREAD

**Discussion by dpriskorn** [WDscholia/scholia#2508] [General]
See discussion here https://en.wikipedia.org/wiki/Wikipedia:Wikipedia_Signpost/2024-05-16/Op-Ed

**Comment by fnielsen** [comment]
With the log that we (as Toolforge "users") can see it is difficult to discern visitors. I think that quite a lot of  the 100,000s hits per day are not "genuine", but bots or one-time clicks.
